tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Exporter.cs

```diff
@@ -61,6 +61,12 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="ExporterSettings.MetricsEndpointUrl"/>
         public const string MetricsEndpointUrl = "SIGNALFX_METRICS_ENDPOINT_URL";
 
+        /// <summary>
+        /// Configuration key for the URL where the Thread Sampler can send stack traces.
```

Comment:
nit. I think that the stack traces are only one use case of logs telemetry. How about something like:

```suggestion
        /// Configuration key for the URL where the logs are exported.
        /// Currently the Thread Sampler sends stack traces as logs.
```